pktoken/mocks/pktoken.go

```diff
@@ -12,12 +12,21 @@ import (
 )
 
 func GenerateMockPKToken(signingKey crypto.Signer, alg jwa.KeyAlgorithm) (*pktoken.PKToken, error) {
+	return GenerateMockPKTokenWithEmail(signingKey, alg, "no-email")
```

Comment:
sorry, this is because of a suggestion I made, but because of the change below, this needs to now be:
```suggestion
	return GenerateMockPKTokenWithEmail(signingKey, alg, "")
```

Reply:
Good catch, fixed!